auto_gptq/modeling/_base.py

```diff
@@ -351,13 +360,45 @@ def tmp(_, inp, out):
                     h.remove()
 
                 for name in subset:
-                    logger.info(f"Quantizing {name} in layer {i + 1}/{len(layers)}...")
-                    scale, zero, g_idx = gptq[name].fasterquant(
-                        percdamp=self.quantize_config.damp_percent,
-                        group_size=self.quantize_config.group_size,
-                        actorder=self.quantize_config.desc_act,
-                        static_groups=self.quantize_config.static_groups,
-                    )
+                    layer_pb.set_description(f"Quantizing {name} in layer {i + 1} of {layer_count}")
+
+                    try:
+                        scale, zero, g_idx, duration, avg_loss = gptq[name].fasterquant(
+                            percdamp=self.quantize_config.damp_percent,
+                            group_size=self.quantize_config.group_size,
+                            actorder=self.quantize_config.desc_act,
+                            static_groups=self.quantize_config.static_groups,
+                        )
+
+                        stat = {"layer": i + 1, "module": name, "avg_loss": avg_loss, "time": duration}
```

Comment:
```suggestion
                        stat = {"layer": i, "module": name, "avg_loss": avg_loss, "time": duration}
```

Reply:
@fxmarty  Can we stick with  `i + 1` so tqdm and layer logs are consistent? using `i + 1` and then bam, having the stat logs printin middle of the log as `i`  and at the very end (summary) is jarring. Also, the final quant_log table is meant for visual consumption more so than code consumption.